pkgnet_build_gallery.R

```diff
@@ -1,8 +1,4 @@
 library(pkgdown)
-library(glue)
```

Comment:
`glue` was introduced at a dependency in #17.  I believe the others were introduced in that same PR.  

[HERE in the vignette](https://github.com/uptake/pkgnet-gallery/blob/master/vignettes/pkgnet-gallery.Rmd#L131) is an example of where `glue` is used. 

I can see how these dependencies could be included within the vignette itself.  

@jameslamb please either move these library() calls to a code chunk at the top of the vignette or leave them where they are today.